src/archiefvernietigingscomponent/destruction/forms.py

```diff
@@ -88,6 +89,8 @@ def save_assignees(self, destruction_list):
                         destruction_list=destruction_list, order=i, assignee=reviewer,
                     )
                 )
+        if assignees:
+            assignees[0].assigned_on = timezone.now()
```

Comment:
Here we are saving the assignees. They have not been assigned yet, so we shouldn't be setting the `assigned_on` here.